solution.rb

```diff
@@ -0,0 +1,39 @@
+TEXT = File.read('speech.txt')
+
+
+class WordCounter
+  @@WORDS_REGEX = /\w+[a-z']*/
```

Comment:
Class instance variables are generally frowned upon. https://github.com/bbatsov/ruby-style-guide#no-class-vars

This should work fine as a constant instead. (Drop the "@@".)